Makefile

```diff
@@ -1,6 +1,8 @@
 .PHONY: build pod manifest manifest-create %-backend %-frontend
 
-GROCY_VERSION = v3.0.1
+GROCY_VERSION = v3.1.0
+COMPOSER_VERSION = 2.1.5
```

Comment:
Thanks for this pull request @HamburgerJungeJr - the `ln` approach to re-routing the `php` binary is nice and straightforward.

Although `composer` 2.1.5 is the latest version currently, I don't think it's required for PHP8 support - any reasons not to use the [Alpine-packaged version](https://pkgs.alpinelinux.org/packages?name=composer&branch=v3.14), currently at 2.1.3?

Reply:
The Alpine package still depends on php7.
I don't know the current state but some time ago it was not so easy to have multiple php versions installed in parallel.

Also in #135 you stated yourself that the php7 dependency prevents the update to 3.1.0. Therefore I removed the alpine-composer dependency to avoid this problem.

Also I don't know if the php-binary at /usr/bin/php is needed by grocy for some internal processing (e.g. cron).
Composer surely needs it, so linking /usr/bin/php to /usr/bin/php8 to to be on the safe side for grocy makes the whole php7 dependency useless as composer now uses php8.

Reply:
That all makes sense, thank you.  I'll aim to do some testing from a build of the updated dockerfile here soon and provide further review / feedback / approval.